nautobot/extras/tasks.py

```diff
@@ -98,7 +98,7 @@ def provision_field(field_id, content_type_pk_set):
 
 
 @nautobot_task
-def process_webhook(webhook_pk, data, model_name, event, timestamp, username, request_id):
+def process_webhook(webhook_pk, data, model_name, event, timestamp, username, request_id, snapshot):
```

Comment:
I noticed that in NetBox's implementation, as well as in your docs update, the key is `snapshots` rather than `snapshot` and that seems more accurate to me anyway - so let's be consistent throughout.

```suggestion
def process_webhook(webhook_pk, data, model_name, event, timestamp, username, request_id, snapshots):
```